torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -768,6 +768,8 @@ ProcessGroupNCCL::ProcessGroupNCCL(
   monitorThreadEnabled_.store(getCvarBool(TORCH_NCCL_ENABLE_MONITORING, true));
   heartbeatTimeoutInSec_ =
       getCvarInt(TORCH_NCCL_HEARTBEAT_TIMEOUT_SEC, 60 * 10 /*10 Mins*/);
+  waitTimeoutDumpInMilSec_ =
```

Comment:
Doc for new env

Reply:
Wait, I think the tradition here is to have the doc for this member in the hpp file?

Reply:
Yea that's fine too